central/namespace/datastore/datastore.go

```diff
@@ -54,11 +46,13 @@ type DataStore interface {
 // New returns a new DataStore instance using the provided store and indexer
 func New(nsStore store.Store, _ graph.Provider, indexer index.Indexer, deploymentDataStore deploymentDataStore.DataStore, namespaceRanker *ranking.Ranker) (DataStore, error) {
 	ds := &datastoreImpl{
-		store:           nsStore,
-		indexer:         indexer,
-		deployments:     deploymentDataStore,
-		namespaceRanker: namespaceRanker,
+		store:             nsStore,
+		indexer:           indexer,
+		deployments:       deploymentDataStore,
+		namespaceRanker:   namespaceRanker,
+		formattedSearcher: formatSearcherV2(indexer, namespaceRanker),
 	}
+
 	ds.formattedSearcher = formatSearcherV2(indexer, namespaceRanker)
```

Comment:
remove?

Reply:
No, there are some wrapper still in use in postgres. But again, focusing on cve datastore in this PR.

Reply:
isn't `formattedSearcher: formatSearcherV2(indexer, namespaceRanker)` and `ds.formattedSearcher = formatSearcherV2(indexer, namespaceRanker)` duplication?

Reply:
I see what you mean